source/atlas-cli-deploy-docker.txt

```diff
@@ -37,43 +37,40 @@ deployment with Docker.
 
       To learn more, see the `Docker documentation <https://docs.docker.com/desktop/install/mac-install/>`__.
 
-   .. step:: Pull down the latest ``mongodb/atlas`` Docker image.
-
-      **Example:**
+   .. step:: Pull down the latest ``mongodb/mongodb-atlas-local`` Docker image.
 
       .. code-block:: sh
 
-         docker pull mongodb/atlas:latest
+         docker pull mongodb/mongodb-atlas-local
 
-   .. step:: Run the Docker image in bash mode.
-
-      To learn more, see :ref:`atlas-cli-docker`.
+   .. step:: Run the Docker image.
 
       **Example:**
 
-      .. code-block:: sh
+      .. tabs::
 
-         docker run -p 27777:27017 --privileged -it mongodb/atlas bash
+         .. tab:: No Authentication
+            :tabid: no-auth
 
-   .. step:: Create a local |service| deployment.
+            .. code-block:: sh
 
-      The following command creates a local |service| deployment with 
-      |fts| capabilities enabled. It also returns a connection string.
+               docker run -p 27017:27017 mongodb/mongodb-atlas-local
 
-      **Example:**
+         .. tab:: With Authentication
+            :tabid: with-auth
 
-      .. code-block::
+            .. code-block:: sh
 
-         atlas deployments setup --bindIpAll --username root --password root --type local --force
+               docker run -e MONGODB_INITDB_ROOT_USERNAME=user MONGODB_INITDB_ROOT_PASSWORD=pass -p 27017:27017 mongodb/mongodb-atlas-local
 
-      To learn more about the available options, see 
-      :ref:`atlas-deployments-setup`.
```

Comment:
Why would it be better to remove "to learn more"?

Reply:
@JuliaMongo, no longer applies with the unified image.